fhir-smart/.gitignore

```diff
@@ -0,0 +1,6 @@
+/.apt_generated/
+/.settings/
+/target/
+/test-output/
+.factorypath
+*.log
```

Comment:
should this perhaps go in a not top-level folder? 
or should this be fhir-auth-smart? 
Is it just going to be scoped to auth intercepting

Reply:
we already have a top-level `.ignore` that covers what we'll need...I'll empty this one out

Reply:
done